### PR TITLE
portalicious: user & roles review

### PR DESCRIPTION
**Pull request by aberonni**
[AB#30630](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/30630)

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

File: interfaces/Portalicious/src/app/components/query-table/query-table.component.html [hunk 1]
```diff
@@ -47,21 +48,26 @@
...
-                class="absolute right-0 top-1/2 -translate-y-1/2"
-                styleClass="bg-transparent hover:text-purple-700"
-              />
+              @if (globalFilterValue()) {
+                <p-button
+                  icon="pi pi-times"
+                  (onClick)="table.filterGlobal(undefined, 'contains')"
+                  text
+                  severity="secondary"
+                  class="absolute right-0 top-1/2 -translate-y-1/2"
```
[the remark this answers is not in the document]
Comment (aberonni) [reply interfaces/Portalicious/src/app/components/query-table/query-table.component.html]: `end-0` seems to do the trick. Not sure what can be done for the vertical alignment.

**Review by elwinschmitz — APPROVED** [review]

File: interfaces/Portalicious/src/app/components/query-table/query-table.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (elwinschmitz) [reply interfaces/Portalicious/src/app/components/query-table/query-table.component.html]: Vertical alignment is not really an issue with translations or anything;
But more the idea that "placing a button vertically centered to the end of the input" sounds like a flex-box thing. And absolute positioning is always a bit of a cheat. ;)

File: interfaces/Portalicious/src/app/components/query-table/query-table.component.html [hunk 2]
```diff
@@ -102,26 +102,29 @@
...
+                  [field]="columnSortField"
                   class="me-1"
                 />
               }
-              @if (!column.disableFiltering) {
+
+              @let columnFilterField = getColumnFilterField(column);
+              @let columnMatchMode = getColumnMatchMode(column);
+
+              @if (columnFilterField) {
```
Comment (elwinschmitz) [inline interfaces/Portalicious/src/app/components/query-table/query-table.component.html:123]: Starts to look a bit like a php-script this file.. :D

Comment (aberonni) [reply interfaces/Portalicious/src/app/components/query-table/query-table.component.html:123]: I know, isn't it glorious? /s

**Pull request merged**